QRCoder/QRCoder.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>net35;net40;netstandard1.3;netstandard2.0;net5.0;net5.0-windows;net6.0;net6.0-windows</TargetFrameworks>
```

Comment:
Do we have to add .NET7 and .NET8 here?

Reply:
No, not unless we are using any .NET 7/8 libraries.  We are not currently doing so.  Personally I only target the minimum number of frameworks needed by the library.  Other developers will target all current TFMs, but really there's no good reason to do so.  It just increases the download size of the nuget package.